pom.xml

```diff
@@ -76,6 +75,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>flatten-maven-plugin</artifactId>
+                <version>1.2.5</version>
```

Comment:
I think it would be better to put this in a separate properties section.

Reply:
Done